### DISCUSSION TITLE
2 wire connection

### DISCUSSION THREAD

**Discussion by ppsirius** [PricelessToolkit/ESPBell-LITE#15] [Q&A]
Hello 
I have only two wire connection, it's old polish analog system codi.
How i should connect ESPBell-MAX?
![20240523_160212](https://github.com/PricelessToolkit/ESPBell-LITE/assets/6945614/343056e4-c9bb-475f-afff-8889cfd8595c)

**Comment by PricelessToolkit** [comment]
Hi, the 2-wire system is not supported, but there might~ be a workaround. You can try connecting the ESPBell bell contacts to the buzzer wires of your intercom if it receives not more than 24V when buzzing "so check it first", and the ESPBell LOCK wires to a button contacts that opens the door. However, first, check if pressing the button opens the door when the intercom handset is in place, or if you need to lift the handset first. If you need to lift the handset to open the door, remote door opening will not work, and you will only receive notifications.
